<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5pt;
	text-indent:-.5pt;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri",sans-serif;
	color:#2F5496;
	font-weight:normal;}
h2
	{mso-style-link:"Heading 2 Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5pt;
	text-indent:-.5pt;
	line-height:107%;
	page-break-after:avoid;
	font-size:13.0pt;
	font-family:"Calibri",sans-serif;
	color:#2F5496;
	font-weight:normal;}
h3
	{mso-style-link:"Heading 3 Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5pt;
	text-indent:-.5pt;
	line-height:107%;
	page-break-after:avoid;
	font-size:12.0pt;
	font-family:"Calibri",sans-serif;
	color:#1F3763;
	font-weight:normal;}
p.MsoToc1, li.MsoToc1, div.MsoToc1
	{margin-top:0in;
	margin-right:3.05pt;
	margin-bottom:4.95pt;
	margin-left:1.25pt;
	text-indent:-.5pt;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:black;}
p.MsoToc2, li.MsoToc2, div.MsoToc2
	{margin-top:0in;
	margin-right:3.05pt;
	margin-bottom:5.05pt;
	margin-left:12.3pt;
	text-indent:-.5pt;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:black;}
p.MsoToc3, li.MsoToc3, div.MsoToc3
	{margin-top:0in;
	margin-right:3.05pt;
	margin-bottom:4.95pt;
	margin-left:23.2pt;
	text-indent:-.5pt;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	color:black;}
a:link, span.MsoHyperlink
	{color:#0563C1;
	text-decoration:underline;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Calibri",sans-serif;
	color:#2F5496;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-link:"Heading 2";
	font-family:"Calibri",sans-serif;
	color:#2F5496;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-link:"Heading 3";
	font-family:"Calibri",sans-serif;
	color:#1F3763;}
.MsoChpDefault
	{font-family:"Calibri",sans-serif;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:107%;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>

</head>

<body lang=EN-US link="#0563C1" vlink="#954F72" style='word-wrap:break-word'>

<div class=WordSection1>

<p class=MsoNormal align=center style='margin-top:0in;margin-right:0in;
margin-bottom:7.95pt;margin-left:7.45pt;text-align:center'><span
style='font-size:44.0pt;line-height:107%'> </span></p>

<p class=MsoNormal align=center style='margin-top:0in;margin-right:0in;
margin-bottom:7.8pt;margin-left:7.45pt;text-align:center'><span
style='font-size:44.0pt;line-height:107%'> </span></p>

<p class=MsoNormal align=center style='margin-top:0in;margin-right:0in;
margin-bottom:7.95pt;margin-left:7.45pt;text-align:center'><span
style='font-size:44.0pt;line-height:107%'> </span></p>

<p class=MsoNormal align=center style='margin-top:0in;margin-right:0in;
margin-bottom:7.95pt;margin-left:7.45pt;text-align:center'><span
style='font-size:44.0pt;line-height:107%'> </span></p>

<p class=MsoNormal align=center style='margin-top:0in;margin-right:31.7pt;
margin-bottom:0in;margin-left:39.1pt;text-align:center;line-height:122%'><span
style='font-size:44.0pt;line-height:122%'>Sig2Lead Ver. 1.1 User Manual </span></p>

<p class=MsoNormal style='margin-bottom:0in'>               </p>

<p class=MsoNormal style='margin-bottom:0in'><span style='font-size:16.0pt;
line-height:107%;color:#2F5496'>Contents </span></p>

<p class=MsoToc1><a href="#_Toc74305756">Sig2Lead Overview<span
style='color:black;display:none;text-decoration:none'>.. </span><span style='color:black;display:none;text-decoration:none'>2</span></a></p>

<p class=MsoToc1><a href="#_Toc74305757">Reference<span style='color:black;
display:none;text-decoration:none'>. </span><span
style='color:black;display:none;text-decoration:none'>2</span></a></p>

<p class=MsoToc2><a href="#_Toc74305758">Paper Abstract<span style='color:black;
display:none;text-decoration:none'> </span><span
style='color:black;display:none;text-decoration:none'>2</span></a></p>

<p class=MsoToc1><a href="#_Toc74305759">Installation of Sig2Lead_v1.1<span
style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>3</span></a></p>

<p class=MsoToc2><a href="#_Toc74305760">Introduction<span style='color:black;
display:none;text-decoration:none'>. </span><span
style='color:black;display:none;text-decoration:none'>3</span></a></p>

<p class=MsoToc2><a href="#_Toc74305761">Installation/Configuration of RStudio
Version<span style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>3</span></a></p>

<p class=MsoToc3><a href="#_Toc74305762">A.  Install R and RStudio<span
style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>3</span></a></p>

<p class=MsoToc3><a href="#_Toc74305763">B.  Download Sig2Lead from Github<span
style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>4</span></a></p>

<p class=MsoToc3><a href="#_Toc74305764">C.  Install Required Libraries<span
style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>4</span></a></p>

<p class=MsoToc2><a href="#_Toc74305765">Installation of Dockerized Version<span
style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>4</span></a></p>

<p class=MsoToc3><a href="#_Toc74305766">A.  Install Docker<span
style='color:black;display:none;text-decoration:none'> </span><span style='color:black;display:none;text-decoration:none'>4</span></a></p>

<p class=MsoToc3><a href="#_Toc74305767">B.  Open Command Prompt<span
style='color:black;display:none;text-decoration:none'> </span><span style='color:black;display:none;text-decoration:none'>4</span></a></p>

<p class=MsoToc3><a href="#_Toc74305768">C.  Download Sig2Lead Container from
Docker Hub<span style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc3><a href="#_Toc74305769">D.  Run Sig2Lead Container<span
style='color:black;display:none;text-decoration:none'> </span><span style='color:black;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc3><a href="#_Toc74305770">E.  Open browser and navigate to localhost:3838<span
style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc3><a href="#_Toc74305771">F.  Run app following instructions
below<span style='color:black;display:none;text-decoration:none'>.. </span><span style='color:black;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc1><a href="#_Toc74305772">Connectivity Analysis Tab<span
style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>6</span></a></p>

<p class=MsoToc2><a href="#_Toc74305773">A.<span style='font-family:"Arial",sans-serif'>
</span>Define Target Gene Workflow<span style='color:black;display:none;
text-decoration:none'>.. </span><span
style='color:black;display:none;text-decoration:none'>6</span></a></p>

<p class=MsoToc2><a href="#_Toc74305774">B.<span style='font-family:"Arial",sans-serif'>
</span>Upload a Signature Workflow<span style='color:black;display:none;
text-decoration:none'>.. </span><span
style='color:black;display:none;text-decoration:none'>8</span></a></p>

<p class=MsoToc2><a href="#_Toc74305775">C.<span style='font-family:"Arial",sans-serif'>
</span>Find Analogs in LINCS Workflow<span style='color:black;display:none;
text-decoration:none'>.. </span><span
style='color:black;display:none;text-decoration:none'>9</span></a></p>

<p class=MsoToc2><a href="#_Toc74305776">D.   Advanced Options<span
style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>10</span></a></p>

<p class=MsoToc1><a href="#_Toc74305777">Chemical Similarity Analysis Tab<span
style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>11</span></a></p>

<p class=MsoToc2><a href="#_Toc74305778">A.  Chemical Similarity Analysis
Options<span style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>12</span></a></p>

<p class=MsoToc1><a href="#_Toc74305779">Network Analysis Tab<span
style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>13</span></a></p>

<p class=MsoToc1><a href="#_Toc74305780">Example Use Case<span
style='color:black;display:none;text-decoration:none'>. </span><span style='color:black;display:none;text-decoration:none'>14</span></a></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:3.9pt;
margin-left:0in;line-height:144%'> </p>

<p class=MsoNormal style='margin-bottom:0in'>               </p>

<h1><a name="_Toc74305756">Sig2Lead Overview</a> </h1>

<p class=MsoNormal style='margin-bottom:7.9pt'> </p>

<p class=MsoNormal style='margin-right:2.2pt'>Sig2Lead aims to facilitate drug
discovery and re-purposing by combining transcriptional signature connectivity
analysis with cheminformatics approaches. In the first step, putative
inhibitors of a target gene specified by the user are identified as those
drug-like molecules in LINCS that have signatures concordant with a KD
signature of the target.  Note that LINCS arguably represents the largest
resource for pharmacogenomics to date, with over 20,000 small molecules and
about 5,000 gene KDs transcriptionally profiled, thus covering a large subset
of the drug-like chemical space and druggable subset of the genome.  In the
second step, a set of additional candidate molecules, e.g., identified by
virtual or experimental screening, can be ranked based on their chemical
similarity to ‘concordant’ LINCS analogs using a fast chemical similarity
search. Furthermore, Sig2Lead can be used to prepare input files for docking
simulations to be performed in conjunction with connectivity-based analysis to
improve the specificity of the search (see paper below), as well as identify
LINCS analogs of user provided compounds irrespective of their connectivity to
a target of interest. </p>

<p class=MsoNormal style='margin-bottom:16.65pt'> </p>

<h1><a name="_Toc74305757">Reference</a> </h1>

<p class=MsoNormal> </p>

<p class=MsoNormal style='margin-right:2.2pt'>Please cite the following paper
that describes the signature connectivity and chemical similarity analyses for
drug discovery implemented in Sig2Lead: </p>

<p class=MsoNormal style='margin-bottom:10.1pt'><a
href="https://www.biorxiv.org/content/10.1101/2020.11.25.399238v1"><span
style='color:#0563C1'>Thorman, A. W., Reigle, J., Chutipongtanate, S.,
Shamsaei, B., Pilarczyk, M., Fazel</span></a><a
href="https://www.biorxiv.org/content/10.1101/2020.11.25.399238v1"><span
style='color:#0563C1'>-</span></a><a
href="https://www.biorxiv.org/content/10.1101/2020.11.25.399238v1"><span
style='color:#0563C1'>Najafabadi, M., ... &amp;</span></a><a
href="https://www.biorxiv.org/content/10.1101/2020.11.25.399238v1"><span
style='color:#0563C1;text-decoration:none'> </span></a><a
href="https://www.biorxiv.org/content/10.1101/2020.11.25.399238v1"><span
style='color:#0563C1'>Meller, J. (2020). Accelerating Drug Discovery and
Repurposing by Combining Transcriptional Signature</span></a><a
href="https://www.biorxiv.org/content/10.1101/2020.11.25.399238v1"><span
style='color:#0563C1;text-decoration:none'> </span></a><a
href="https://www.biorxiv.org/content/10.1101/2020.11.25.399238v1"><span
style='color:#0563C1'>Connectivity with Docking.</span></a><a
href="https://www.biorxiv.org/content/10.1101/2020.11.25.399238v1"><span
style='color:#0563C1'> </span></a><a
href="https://www.biorxiv.org/content/10.1101/2020.11.25.399238v1"><i><span
style='color:#0563C1'>bioRxiv</span></i></a><a
href="https://www.biorxiv.org/content/10.1101/2020.11.25.399238v1"><span
style='color:#0563C1'>.</span></a><a
href="https://www.biorxiv.org/content/10.1101/2020.11.25.399238v1"><span
style='color:#0563C1;text-decoration:none'> </span></a></p>

<h2><a name="_Toc74305758">Paper Abstract</a> </h2>

<p class=MsoNormal style='margin-top:0in;margin-right:2.3pt;margin-bottom:7.9pt;
margin-left:0in;text-align:justify'>The development of targeted treatment
options for precision medicine is hampered by a slow and costly process of drug
screening. While small molecule docking simulations are often applied in
conjunction with cheminformatic methods to reduce the number of candidate
molecules to be tested experimentally, the current approaches suffer from high
false positive rates and are computationally expensive. Here, we present a
novel <i>in silico</i> approach for drug discovery and repurposing, dubbed <i>connectivity
enhanced</i> Structure Activity Relationship (<i>ce</i>SAR) that improves on
current methods by combining docking and virtual screening approaches with
pharmacogenomics and transcriptional signature connectivity analysis. <i>ce</i>SAR
builds on the landmark LINCS library of transcriptional signatures of over
20,000 drug-like molecules and ~5,000 gene knock-downs (KDs) to connect small
molecules and their potential targets. For a set of candidate molecules and
specific target gene, candidate molecules are first ranked by chemical similarity
to their ‘concordant’ LINCS analogs that share signature similarity with a
knock-down of the target gene. An efficient method for chemical similarity
search, optimized for sparse binary fingerprints of chemical moieties, is used
to enable fast searches for large libraries of small molecules. A small subset
of candidate compounds identified in the first step is then re-scored by
combining signature connectivity with docking simulations. On a set of 20 DUD-E
benchmark targets with LINCS KDs, the consensus approach reduces significantly
false positive rates, improving the median precision 3-fold over docking
methods at the extreme library reduction. We conclude that signature
connectivity and docking provide complementary signals, offering an avenue to
improve the accuracy of virtual screening while reducing run times by multiple
orders of magnitude. </p>

<p class=MsoNormal> </p>

<p class=MsoNormal style='margin-bottom:16.65pt'> </p>

<h1><a name="_Toc74305759">Installation of Sig2Lead_v1.1</a> </h1>

<p class=MsoNormal style='margin-bottom:9.95pt'> </p>

<h2><a name="_Toc74305760">Introduction</a> </h2>

<p class=MsoNormal> </p>

<p class=MsoNormal style='margin-right:2.2pt'>Sig2Lead is available as a Shiny
app to be executed in RStudio or as a docker container to be executed in a
system-independent manner.  The RStudio version is available on Github and the
dockerized version is available on Docker Hub.  The RStudio version requires
that R and RStudio be locally installed on the user’s computer and requires the
installation of all requisite R packages.  The dockerized version requires installation
of docker and Sig2Lead image.  Once the Sig2Lead image is running, a web
browser is used to run the app without the need to install R, RStudio, or all
necessary R packages.  This manual provides step-by-step instructions for both
versions.   </p>

<p class=MsoNormal style='margin-bottom:9.95pt'> </p>

<h2><a name="_Toc74305761">Installation/Configuration of RStudio Version</a> </h2>

<p class=MsoNormal style='margin-bottom:8.85pt'> </p>

<h3><a name="_Toc74305762">A.  Install R and RStudio</a> </h3>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:18.5pt'>Sig2Lead was built with R version 4.0.3 and is expected to
run with any later versions of R (to avoid R dependencies please see dockerized
version).  The latest version of R can be downloaded at: </p>

<p class=MsoNormal style='margin-bottom:7.9pt'>              <a
href="http://www.r-project.org/"><span style='color:#0563C1'>http://www.r</span></a><a
href="http://www.r-project.org/"><span style='color:#0563C1'>-</span></a><a
href="http://www.r-project.org/"><span style='color:#0563C1'>project.org/</span></a><a
href="http://www.r-project.org/"><span style='color:windowtext;text-decoration:
none'> </span></a></p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:0in;
margin-left:18.5pt;line-height:172%'>Additionally, RStudio is required and can
be downloaded at:   <a href="https://www.rstudio.com/products/rstudio/download/"><span
style='color:#0563C1'>https://www.rstudio.com/products/rstudio/download/</span></a><a
href="https://www.rstudio.com/products/rstudio/download/"><span
style='color:windowtext;text-decoration:none'> </span></a></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:8.95pt;
margin-left:.25in'> </p>

<h3><a name="_Toc74305763">B.  Download Sig2Lead from Github</a> </h3>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:18.5pt'>Sig2Lead and associated files can be downloaded from: </p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:7.9pt;
margin-left:17.75pt'><a href="https://github.com/sig2lead/sig2lead_v1.1/">https://github.com/sig2lead/sig2lead_v1.1/</a><a
href="https://github.com/sig2lead/sig2lead_v1/"><span style='color:windowtext;
text-decoration:none'> </span></a></p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:18.5pt'>Navigate to the Sig2Lead_v1 repository, click on “Code”,
and select “Download Zip”.  The downloaded zip file needs to be unzipped to a
user-selected directory. </p>

<h3><a name="_Toc74305764">C.  Install Required Libraries</a> </h3>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:18.5pt'>Once R and RStudio are installed, shiny must be installed.
This can be completed by typing into the R console: </p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.25in'><b>install.packages(“shiny”) </b> </p>

<p class=MsoNormal style='margin-right:2.2pt'>All other dependencies and
libraries will be installed upon the first time running the application. </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:5.8pt;
margin-left:0in'>This step may not be handled properly on MacOS, requiring a
step-by-step installation of missing libraries. An alternative for Mac users is
to use the dockerized version that takes care of all the dependencies. </p>

<p class=MsoNormal style='margin-right:2.2pt'>To run the application, click
the  <img border=0 width=98 height=28 id="Picture 796"
src="sig2Lead_version_1_1_user_manual_for_help_rev_files/image001.jpg">  button
at the top middle of the RStudio interface. </p>

<p class=MsoNormal style='margin-bottom:10.05pt'> </p>

<h2><a name="_Toc74305765">Installation of Dockerized Version</a> </h2>

<p class=MsoNormal style='margin-bottom:8.85pt'> </p>

<h3><a name="_Toc74305766">A.  Install Docker</a> </h3>

<p class=MsoNormal style='margin-bottom:7.9pt'> </p>

<p class=MsoNormal style='margin-bottom:0in'><span style='color:#24292E'>Ubuntu:
follow</span><a
href="https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/"><span
style='color:#24292E;text-decoration:none'> </span></a><a
href="https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/"><span
style='color:blue;text-decoration:none'>the instructions</span></a><a
href="https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/"><span
style='color:#24292E;text-decoration:none'> </span></a><span style='color:#24292E'>to
get Docker CE for Ubuntu. </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:76.05pt;margin-bottom:
0in;margin-left:0in;line-height:99%'><span style='color:#24292E'>Mac: follow</span><a
href="https://store.docker.com/editions/community/docker-ce-desktop-mac"><span
style='color:#24292E;text-decoration:none'> </span></a><a
href="https://store.docker.com/editions/community/docker-ce-desktop-mac"><span
style='color:blue;text-decoration:none'>the instructions</span></a><a
href="https://store.docker.com/editions/community/docker-ce-desktop-mac"><span
style='color:#24292E;text-decoration:none'> </span></a><span style='color:#24292E'>to
install</span><a href="https://download.docker.com/mac/stable/Docker.dmg"><span
style='color:#24292E;text-decoration:none'> </span></a><a
href="https://download.docker.com/mac/stable/Docker.dmg"><span
style='color:blue;text-decoration:none'>the Stable version of Docker CE</span></a><a
href="https://download.docker.com/mac/stable/Docker.dmg"><span
style='color:#24292E;text-decoration:none'> </span></a><span style='color:#24292E'>on
Mac. Windows: follow</span><a
href="https://docs.docker.com/toolbox/toolbox_install_windows/"><span
style='color:#24292E;text-decoration:none'> </span></a><a
href="https://docs.docker.com/toolbox/toolbox_install_windows/"><span
style='color:blue;text-decoration:none'>the instructions</span></a><a
href="https://docs.docker.com/toolbox/toolbox_install_windows/"><span
style='color:#24292E;text-decoration:none'> </span></a><span style='color:#24292E'>to
install</span><a href="https://download.docker.com/win/stable/DockerToolbox.exe"><span
style='color:#24292E;text-decoration:none'> </span></a><a
href="https://download.docker.com/win/stable/DockerToolbox.exe"><span
style='color:blue;text-decoration:none'>Docker Toolbox</span></a><a
href="https://download.docker.com/win/stable/DockerToolbox.exe"><span
style='color:#24292E;text-decoration:none'> </span></a><span style='color:#24292E'>on
Windows. </span></p>

<p class=MsoNormal style='margin-bottom:0in'><span style='color:#24292E'> </span></p>

<p class=MsoNormal style='margin-right:2.2pt'>For list of useful docker
commands please see:<b> </b></p>

<p class=MsoNormal style='margin-bottom:12.0pt;line-height:normal'><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>https://www.digitalocean.com/community/tutorials/how</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>-</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>to</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>-</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>remove</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>-</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>docker</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>-</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>images</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>containers</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>-</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>and</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>-</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:blue'>volumes</span></a><a
href="https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes"><span
style='font-size:12.0pt;color:#24292E;text-decoration:none'> </span></a></p>

<h3><a name="_Toc74305767">B.  Open Command Prompt</a> </h3>

<p class=MsoNormal style='margin-bottom:8.05pt'> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:.05pt;
margin-left:0in'>Open a command prompt using Powershell for Windows machines or
a terminal window for Macs.  On </p>

<p class=MsoNormal style='margin-bottom:7.9pt'>Windows machine, click on the
Start Menu, click on the “Windows Powershell” folder and tap “Windows
PowerShell”.  An alternative method is to go to the search bar in the lower
left corner of the screen and input “Powershell”, which will display a menu option
to choose and open “Windows Powershell”.  On Mac, launch Spotlight Search by
clicking on the magnifying glass icon in the menu bar (or press
Command+Space).  When the Spotlight Search bar pops up on your screen, type
“terminal.app” and hit Return.  This will open a terminal. </p>

<p class=MsoNormal style='margin-bottom:8.95pt'> </p>

<h3><a name="_Toc74305768">C.  Download Sig2Lead Container from Docker Hub</a> </h3>

<p class=MsoNormal style='margin-bottom:7.9pt'>  </p>

<p class=MsoNormal style='margin-right:2.2pt'>To download the docker image run
the following command in a command prompt:      </p>

<p class=MsoNormal style='margin-left:35.75pt'><b>docker pull
reiglej/sig2lead:v1_1 </b></p>

<p class=MsoNormal style='margin-right:2.2pt'>Linux users may need to use sudo
to run Docker by typing: <b>sudo docker pull reiglej/sig2lead:v1_1</b> <b> </b></p>

<p class=MsoNormal style='margin-right:2.2pt'>Note that the Sig2Lead image is
available from Docker Hub at <a
href="http://hub.docker.com/repositories/Sig2Lead/"><span style='color:#0563C1'>http://hub.docker.com/repositories/Sig2Lead/</span></a><a
href="http://hub.docker.com/repositories/Sig2Lead/"><span style='color:windowtext;
text-decoration:none'> </span></a> </p>

<p class=MsoNormal style='margin-bottom:8.85pt'> </p>

<h3><a name="_Toc74305769">D.  Run Sig2Lead Container</a>  </h3>

<p class=MsoNormal style='margin-left:15.0pt'> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:15.5pt'>To run Sig2Lead container, open a command prompt and run
following command:  </p>

<p class=MsoNormal style='margin-left:35.75pt'><b>docker run -p 3838:3838 -v <i>&lt;local_path&gt;</i>:/srv/shiny-server/userfile
<i>-</i>d<i> </i>reiglej/sig2lead:v1_1</b> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:12.85pt'>The &lt;local path&gt; is the path of the local directory
that contains the smiles or sdf file of your added compounds.   </p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:7.9pt;
margin-left:12.85pt'>For Windows, the following line is an example with local
directory “C:\Added_Compounds” </p>

<p class=MsoNormal style='margin-left:35.75pt'><b>docker run -p 3838:3838 -v 
C:\Added_Compounds:/srv/shiny-server/userfile -d reiglej/sig2lead:v1_1 </b></p>

<p class=MsoNormal align=center style='margin-right:16.2pt;text-align:center'>For
MAC/UNIX, the following line is an example with local directory
“/User/Added_Compounds” </p>

<p class=MsoNormal style='margin-left:35.75pt'><b>docker run -p 3838:3838 -v 
/User/Added_Compounds:/srv/shiny-server/userfile -d reiglej/sig2lead:v1_1</b> </p>

<p class=MsoNormal style='margin-right:2.2pt'>Before running the container make
sure that port 3838 is free to run.   You can stop and kill any other docker
containers on this port by </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:36.5pt'>[sudo] docker stop &lt;container ID&gt; &amp;&amp; docker
rm &lt;container ID&gt; </p>

<p class=MsoNormal style='margin-right:2.2pt'>To check the container ID run
this command: </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:36.5pt'>docker ps -a </p>

<h3><a name="_Toc74305770">E.  Open browser and navigate to localhost:3838</a> </h3>

<p class=MsoNormal style='margin-right:2.2pt'>      Open browser and navigate
to the following url: <u>http://127.0.0.1:3838</u> </p>

<h3><a name="_Toc74305771">F.  Run app following instructions below</a> </h3>

<p class=MsoNormal style='margin-right:2.2pt'>    Follow the instructions below
for running the app. </p>

<p class=MsoNormal style='margin-bottom:16.65pt'> </p>

<h1><a name="_Toc74305772">Connectivity Analysis Tab</a> </h1>

<p class=MsoNormal style='margin-bottom:7.9pt'> </p>

<p class=MsoNormal style='margin-right:2.2pt'>In the next several sections,
several distinct workflows organized in 3 different tabs are discussed and
illustrated using use cases. The first of those sections below describes the
main tab where the primary workflow can be initiated by defining the target
gene. </p>

<p class=MsoNormal style='margin-bottom:12.0pt'> </p>

<h2><a name="_Toc74305773">A.</a><span style='font-family:"Arial",sans-serif'> </span>Define
Target Gene Workflow </h2>

<p class=MsoNormal style='margin-bottom:0in'> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:18.5pt'>This is the standard workflow for identifying small
molecule inhibitors or activators of a target of interest. Within this
workflow, a gene of interest is required and optionally, a user-defined list of
candidate compounds for scoring can be provided in SDF or SMILES format.
Sig2Lead collects data from genetic knockdowns within LINCS and identifies
compounds that generate highly concordant transcriptional signatures to these
genetic knockdowns, i.e., putative target/pathway inhibitors. That step is
dependent on ilincs.org API calls and requires Internet connection.  </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:18.5pt'>If an external set of candidate molecules is provided,
e.g., identified by virtual or experimental screening, these user-provided
candidate molecules are ranked based on their chemical similarity to ‘concordant’
LINCS analogs using a fast chemical similarity search. This method allows
scoring of small molecules to be tested for the purpose of library reduction.
Results for user added compounds will be ranked in descending order of
similarity to ‘concordant’ LINCS analogs, with ties broken by concordance
scores in the table titled “My Candidates Ranked”.  Candidate compounds
identified from within the LINCS library as concordant to a target KD will be
scored only by their concordance scores (obtained from ilincs.org) to a
knockdown of the target gene of interest in the table titled “LINCS Compounds
Ranked”.  These tables can be downloaded using the download buttons just below
the “Go!” button. See Figures 1 and 2 below. </p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:5.8pt;
margin-left:.25in'> </p>

<p class=MsoNormal align=right style='margin-bottom:4.9pt;text-align:right'><img
border=0 width=624 height=388 id="Picture 1087"
src="sig2Lead_version_1_1_user_manual_for_help_rev_files/image002.jpg"> </p>

<p class=MsoNormal style='margin-right:2.2pt'><b>Figure 1</b>:  Define Target
Gene workflow.  In this primary workflow, the user defines a gene of interest
to identify putative pathway/target inhibitors. If the user defines a set of
compounds from some other analyses, those compounds will be compared to the
LINCS library for analogous compounds and scored by similarity and concordance
in the upper table (My Candidates Ranked), which will only appear in the event
of added compounds. The lower table (LINCS Candidates Ranked) will always
appear when running this workflow and scores all LINCS compounds with a
concordance above the threshold in descending order of concordance. </p>

<p class=MsoNormal style='margin-bottom:5.8pt'> </p>

<p class=MsoNormal align=right style='margin-top:0in;margin-right:16.2pt;
margin-bottom:4.9pt;margin-left:0in;text-align:right'><img border=0 width=602
height=370 id="Picture 1175"
src="sig2Lead_version_1_1_user_manual_for_help_rev_files/image003.jpg"> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:.25pt;margin-bottom:10.0pt;
margin-left:0in;line-height:99%'><b>Figure 2:<i><span style='color:#44546A'> </span></i></b><span
style='color:#44546A'>The two file input controls for the RStudio version
(left) and the dockerized version (right) are shown in the above figure.  The
RStudio version (left) has a standard upload control in which the user </span><span
style='color:#44546A'>can press the “Browse” button and navigate to the
directory with the “</span><span style='color:#44546A'>.smi</span><span
style='color:#44546A'>”</span><span style='color:#44546A'> or </span><span
style='color:#44546A'>“</span><span style='color:#44546A'>.sdf </span><span
style='color:#44546A'>“</span><span style='color:#44546A'>files of userprovided
compounds.  The file input control for the dockerized version (right) is
slightly different with a </span><span style='color:#44546A'>button labeled
“Select from My Files”</span><span style='color:#44546A'> which when pressed
lists </span><span style='color:#44546A'>”</span><span style='color:#44546A'>.smi</span><span
style='color:#44546A'>”</span><span style='color:#44546A'>,</span><span
style='color:#44546A'>”</span><span style='color:#44546A'>.txt</span><span
style='color:#44546A'>”</span><span style='color:#44546A'>, and </span><span
style='color:#44546A'>“</span><span style='color:#44546A'>.sdf</span><span
style='color:#44546A'>”</span><span style='color:#44546A'> files in the </span><span
style='color:#44546A'>user’s </span><span style='color:#44546A'>local mounted
directory (instructions for mounting a directory when launching the dockerized
version are described in </span><span style='color:#44546A'>the ‘Installation
of Dockerized Version’</span><span style='color:#44546A'> section) which the
user selects. </span></p>

<p class=MsoNormal> </p>

<p class=MsoNormal style='margin-bottom:11.9pt'> </p>

<h2><a name="_Toc74305774">B.</a><span style='font-family:"Arial",sans-serif'> </span>Upload
a Signature Workflow </h2>

<p class=MsoNormal style='margin-bottom:7.9pt'> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:18.5pt'>This workflow allows users to upload a signature of their
own to search for potential inhibitors to a target gene unavailable on LINCS,
or molecules that generate a similar signature to some other system
perturbation that is otherwise undefined. In this workflow, users define a
signature using one of the formats defined at (<a
href="http://www.ilincs.org/ilincs/signatures/main/"><span style='color:#0563C1'>www.ilincs.org/ilincs/signatures/main/</span></a><a
href="http://www.ilincs.org/ilincs/signatures/main/"><span style='color:windowtext;
text-decoration:none'>)</span></a>. This pipeline otherwise follows the same
pipeline as the “Define Target Gene” workflow. </p>

<p class=MsoNormal style='margin-bottom:0in'> </p>

<p class=MsoNormal align=right style='margin-bottom:4.9pt;text-align:right'><img
border=0 width=624 height=361 id="Picture 1243"
src="sig2Lead_version_1_1_user_manual_for_help_rev_files/image004.jpg"> </p>

<p class=MsoNormal style='margin-right:2.2pt'><b>Figure 3.  </b> Upload a
Signature Workflow.  This workflow allows one to query iLINCS (see ilincs.org)
to find small molecules in LINCS that have signatures concordant with a user
provided transcriptional signature.  The signature file should be saved as a
tab-separated text file with gene symbol, log of differential expression value,
and p-value comprising the columns in the that order.  The output of this
workflow is a table of ‘concordant’ LINCS small molecules that have
transcriptional signatures similar to the uploaded signature. </p>

<p class=MsoNormal style='margin-bottom:12.0pt'> </p>

<h2><a name="_Toc74305775">C.</a><span style='font-family:"Arial",sans-serif'> </span>Find
Analogs in LINCS Workflow </h2>

<p class=MsoNormal> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:18.5pt'>This workflow simply identifies LINCS analogs to
user-defined compounds, irrespective of their connectivity to a target. This
requires added compounds and is useful in determining if there are
transcriptionally profiled analogs in the LINCS library. LINCS compounds are
drug or drug-like, many of them with known Mode of Action (MOA), so this may
also be used as a simple filter to remove compounds that do not contain normal
drug-like structures. Additionally, it was used in benchmarking as a baseline
similarity of various compound libraries to the LINCS small molecules (see the
reference paper). </p>

<p class=MsoNormal style='margin-bottom:0in'> </p>

<p class=MsoNormal align=right style='margin-top:0in;margin-right:9.85pt;
margin-bottom:5.0pt;margin-left:0in;text-align:right'><img border=0 width=611
height=299 id="Picture 1400"
src="sig2Lead_version_1_1_user_manual_for_help_rev_files/image005.jpg"> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:6.15pt;
margin-left:0in'><b>Figure 4</b>:  Find Analogs in LINCS Workflow.  This
workflow allows the user to find small molecules included in LINCS library that
are structurally similar to user-provided compounds.  The Advanced Options can
be used to change the number of LINCS analogs to be returned for each
user-provided compound, bounded by the chemical similarity threshold (also
adjustable).  The user can also use either the ultrafast <i>minSim</i> algorithm
or slower <i>fpSim</i> algorithm to compute chemical similarity. </p>

<p class=MsoNormal style='margin-bottom:12.95pt'><i><span style='font-size:
9.0pt;line-height:107%;color:#44546A'> </span></i></p>

<h2><a name="_Toc74305776">D.   Advanced Options</a> </h2>

<p class=MsoNormal style='margin-bottom:9.6pt'> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:1.6pt;
margin-left:.5in;text-indent:-.25in;line-height:107%'><span style='color:black'>A.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>Users
can select options to identify either inhibitors or activators. Functionally,
selecting activators results in searching for molecules that are discordant to
the gene knockdown of interest instead of concordant. </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:1.6pt;
margin-left:.5in;text-indent:-.25in;line-height:107%'><span style='color:black'>B.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>Users
can change the method used for similarity search. The default, minSim is an
exact fast chemical similarity search utilized by Sig2Lead introduced in the
reference paper. For comparison, an established fpSim function to compute
chemical similarity is available as a slow option (it typically runs about 100x
slower than minSim). </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:.5in;text-indent:-.25in;line-height:107%'><span style='color:black'>C.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>Finally,
users can select the version of the iLINCS database to query.  The choices are
“Current” which is the latest version of iLINCS or the “Legacy” version which
is the previous version.  The concordance threshold can be changed also.  The
default is 0.1.  For “Current“ iLINCS, the minimum concordance value observed
is 0.162, while for “Legacy” iLINCS the minimum concordance threshold observed
is 0.2.   For some targets, a more stringent cutoff may be required to increase
the specificity. </p>

<p class=MsoNormal align=center style='margin-top:0in;margin-right:102.05pt;
margin-bottom:4.9pt;margin-left:0in;text-align:center'><img border=0 width=488
height=298 id="Picture 1476"
src="sig2Lead_version_1_1_user_manual_for_help_rev_files/image006.jpg"> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:6.15pt;
margin-left:0in'><b>Figure 5</b>. Define Target Gene Workflow Advanced
Options.  </p>

<p class=MsoNormal style='margin-bottom:10.95pt'><i><span style='font-size:
9.0pt;line-height:107%;color:#44546A'> </span></i></p>

<p class=MsoNormal style='margin-bottom:16.75pt'> </p>

<h1><a name="_Toc74305777">Chemical Similarity Analysis Tab</a> </h1>

<p class=MsoNormal style='margin-bottom:7.9pt'> </p>

<p class=MsoNormal style='margin-right:2.2pt'>After searching LINCS for ‘concordant’
analogs of user provided candidate compounds, both LINCS and user provided compounds
can be further analyzed through chemical similarity using the “Chemical
Similarity Analysis” tab. To run chemical similarity analysis click the “Run
SAR” button. This will initiate a clustering analysis that compares concordant
compounds from LINCS for the target and user-added compounds and clusters them
by chemical similarity to one another.  By default, 5,000 compounds will be
analyzed (this can be changed under advanced options if an extended time is
acceptable). The output will be in the form of two figures, a heatmap and an
MDS plot, and a table of centroids for each cluster. The heatmap is generated
through hierarchical clustering and shows a distance matrix comparing each
compound identified through LINCS (Green) or added by the user (Blue). The MDS
plot is an alternative view and implicit clustering by projection into 2D,
showing relative distances between clusters of compounds. The radius of each
pie chart corresponds to the size of the cluster. </p>

<p class=MsoNormal align=right style='margin-bottom:5.0pt;text-align:right'><img
border=0 width=624 height=390 id="Picture 1533"
src="sig2Lead_version_1_1_user_manual_for_help_rev_files/image007.jpg"> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:.05pt;
margin-left:0in'><b>Figure 6.  </b>Example of chemical similarity analysis. 
SAR type analyses can be performed for either </p>

<p class=MsoNormal style='margin-right:2.2pt'>‘concordant’ LINCS compounds
(here shown in Green), or user provided candidate molecules (here in Blue), or
together to identify distinct classes of chemical moieties within the set of
user-provided compounds and their ‘concordant’ LINCS analogs.  The heatmap in
the left panel shows pairwise chemical similarity pattern with the compounds
analyzed as both columns and rows, i.e., the diagonal represents identity
(Tanimoto coefficient of 1.0).  Note that user defined candidates indicated by
blue ticks in the top bar are scattered throughout, i.e., they belong to
several distinct classes of compounds with most of them in the middle ‘big’ cluster
indicated by a large rectangular block of high similarity scores in the middle
of the heatmap.  This is further highlighted in the right panel that shows
individual clusters identified using the MDS 2-dimensional projection of
pairwise similarities, with the ‘big’ circle in the bottom left corner
corresponding to the central mixed cluster in the heatmap. </p>

<p class=MsoNormal style='margin-bottom:9.95pt'> </p>

<h2><a name="_Toc74305778">A.  Chemical Similarity Analysis Options</a> </h2>

<p class=MsoNormal style='margin-bottom:9.6pt'> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:.05pt;
margin-left:.5in;text-indent:-.25in;line-height:107%'><span style='color:black'>A.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>For
clustering, different Tanimoto similarity thresholds can be selected. The
default is 0.75, which groups compounds of modest difference together, but can
be adjusted to make more </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:1.75pt;
margin-left:36.5pt'>(closer to 1) or less stringent clusters (closer to 0).
This will only be reflected in the MDS plot and the table of representatives. </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:.05pt;
margin-left:.5in;text-indent:-.25in;line-height:107%'><span style='color:black'>B.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>The
minimum cluster size for inclusion in the MDS plot can also be changed. The
user can cluster all compounds regardless of cluster size or increase this
threshold to only consider those from large clusters of related compounds. </p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:0in;
margin-left:.5in'> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:5.75pt;
margin-left:.5in'> </p>

<p class=MsoNormal align=center style='margin-top:0in;margin-right:107.8pt;
margin-bottom:4.9pt;margin-left:0in;text-align:center'><img border=0 width=480
height=298 id="Picture 1685"
src="sig2Lead_version_1_1_user_manual_for_help_rev_files/image008.jpg"> </p>

<p class=MsoNormal style='margin-right:2.2pt'><b>Figure 7.  </b>Chemical
Similarity Analysis Advanced Options </p>

<p class=MsoNormal style='margin-bottom:16.65pt'> </p>

<h1><a name="_Toc74305779">Network Analysis Tab</a> </h1>

<p class=MsoNormal style='margin-bottom:8.05pt'> </p>

<p class=MsoNormal style='margin-right:2.2pt'>Finally, Network connectivity
analyses can be performed after chemical similarity analyses using STITCH on
the “Network Analysis” tab. This analysis is intended to scrape any known
information about identified compounds and their interactions with members of
the pathway of interest. This step can be performed either through a global
view (all identified compounds) or on a cluster-by-cluster basis (much
faster).    </p>

<p class=MsoNormal style='margin-right:2.2pt'>For cluster-by-cluster analysis, select
“Cluster Network STITCH” from the drop-down box.  Following this click the “View
Selected Cluster Network” button to see the gene-chemical interaction network
of the specific cluster number (by default, the cluster number is set to 1) of
the MDS plot from the “Chemical Similarity Analysis” tab (Figure 7).  </p>

<p class=MsoNormal style='margin-right:2.2pt'>When selecting “Global STITCH”
from the drop-down box, all compounds found in clusters of sufficient size, as
determined from the “Chemical Similarity Analysis” tab’s advanced options, will
be included in the network analysis.  If the “Shows all clusters” box is
checked, all compounds are included regardless of the size of the cluster of
which they are in. The Global STITCH analysis can be slow, depending on the
number of compounds added.   </p>

<p class=MsoNormal style='margin-right:2.2pt'>This network analysis can help
the user map the added compounds or the LINCS analogs of the selected cluster
as target or pathway inhibitors. As an example, target inhibitors that directly
interact with BCL2A1 as the target gene are shown in Figure 8. The molecules
included are the user-added compounds and their LINCS chemical analogs based on
the MDS clustering for the selected cluster.  </p>

<p class=MsoNormal style='margin-bottom:7.9pt'> </p>

<p class=MsoNormal style='margin-bottom:5.8pt'> </p>

<p class=MsoNormal align=right style='margin-bottom:4.9pt;text-align:right'><img
border=0 width=624 height=373 id="Picture 1761"
src="sig2Lead_version_1_1_user_manual_for_help_rev_files/image009.jpg"> </p>

<p class=MsoNormal style='margin-right:2.2pt'><b>Figure 8</b>: STITCH network
analysis for BCL2A1 (the target gene showed as a yellow node) and associated
chemicals, with the user-added compounds and their ‘concordant’ LINCS similar
analogs showed as the magenta nodes. The edge between nodes represents their
corresponding interactions based on the experimental evidence and text mining
retrieved from the STITCH database. </p>

<p class=MsoNormal style='margin-bottom:16.65pt'> </p>

<h1><a name="_Toc74305780">Example Use Case</a> </h1>

<p class=MsoNormal style='margin-bottom:7.9pt'> </p>

<p class=MsoNormal style='margin-right:2.2pt'>This use case demonstrates the
screening for putative inhibitors of EGFR with a set of user provided candidate
compounds.  </p>

<p class=MsoNormal style='margin-bottom:9.6pt'> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:9.6pt;
margin-left:.5in;text-indent:-.25in'><span style='color:black'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>In
the “Input a Target Gene” box, type EGFR  </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:.05pt;
margin-left:.5in;text-indent:-.25in;line-height:107%'><span style='color:black'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>For
&quot;Added candidate compounds&quot;, this demo will use the known EGFR active
ligands downloaded from the DUD-E database </p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:9.6pt;
margin-left:36.5pt'><a
href="http://dude.docking.org/targets/egfr/actives_final.sdf.gz"><span
style='color:windowtext;text-decoration:none'>(</span></a><a
href="http://dude.docking.org/targets/egfr/actives_final.sdf.gz"><span
style='color:#0563C1'>http://dude.docking.org/targets/egfr/actives_final.sdf.gz</span></a><a
href="http://dude.docking.org/targets/egfr/actives_final.sdf.gz"><span
style='color:windowtext;text-decoration:none'>)</span></a>. Please browse to
upload “EGFR_actives_final.sdf” which is already provided in the Sig2Lead
folder.  </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:.5in;text-indent:-.25in;line-height:107%'><span style='color:black'>3.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>Click
the advance options button. The users will see three options show up here.
Please choose “Legacy”, &quot;Inhibit&quot;, &quot;minSim”, and “0.3” for these
options (as shown in Figure 9), and then click Go! </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:.5in;text-indent:-.25in;line-height:107%'><span style='color:black'>4.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>Once
the analysis is finished, two tables will show up. The first table contains the
similarity scores between the added compounds and their ‘concordant’ LINCS
analog (the last column). The similarity of 1 for a user provided compound
indicates that it was in fact included in the LINCS library and directly profiled.
The second table shows ranking of LINCS candidates as potential EGFR
targeted/pathway inhibitors based on their concordance score. </p>

<p class=MsoNormal style='margin-bottom:5.8pt'> </p>

<p class=MsoNormal align=right style='margin-top:0in;margin-right:16.55pt;
margin-bottom:4.9pt;margin-left:0in;text-align:right'><img border=0 width=601
height=374 id="Picture 1883"
src="sig2Lead_version_1_1_user_manual_for_help_rev_files/image010.jpg"> </p>

<p class=MsoNormal style='margin-bottom:7.9pt'> </p>

<p class=MsoNormal style='margin-right:2.2pt'><b>Figure 9</b>. Signature
Connectivity Analysis using EGFR as the target gene.  </p>

<p class=MsoNormal style='margin-bottom:7.9pt'> </p>

<p class=MsoNormal style='margin-bottom:9.7pt'> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:9.6pt;
margin-left:.5in;text-indent:-.25in'><span style='color:black'>5.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>Next,
go to the “Chemical Similarity” tab and click “Run SAR” button. This step may
take a while.  </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:9.8pt;
margin-left:.5in;text-indent:-.25in;line-height:107%'><span style='color:black'>6.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>Once
finished, a heatmap and a multidimensional scaling plot will be generated
(Figure 10). Both plots represent the added compounds in blue and LINCS
compounds in green. These compounds are present in the same cluster based on
their chemical similarity. To change the chemical similarity threshold for this
analysis, please click &quot;Show Advanced Options&quot; button. </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:.5in;text-indent:-.25in;line-height:107%'><span style='color:black'>7.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>A
table below the plots contains the representatives (centroids) of each MDS
cluster. To download the full table result, please click “Representatives”
button. Here, the users can retrieve LINCS compounds that may share the same
activity as the user-added candidate compounds. </p>

<p class=MsoNormal style='margin-bottom:7.9pt'> </p>

<p class=MsoNormal> </p>

<p class=MsoNormal style='margin-bottom:5.75pt'> </p>

<p class=MsoNormal align=right style='margin-top:0in;margin-right:16.55pt;
margin-bottom:5.05pt;margin-left:0in;text-align:right'><img border=0 width=601
height=372 id="Picture 1962"
src="sig2Lead_version_1_1_user_manual_for_help_rev_files/image011.jpg"> </p>

<p class=MsoNormal style='margin-bottom:7.9pt'> </p>

<p class=MsoNormal style='margin-right:2.2pt'><b>Figure 10</b>. Chemical
similarity analysis of user-added compounds against LINCS chemicals. Blue and
green colors are used to label the user-added compounds and LINCS analogs,
respectively, in the heatmap (left) and the MDS plot (right). </p>

<p class=MsoNormal style='margin-bottom:9.7pt'> </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:9.7pt;
margin-left:.5in;text-indent:-.25in;line-height:107%'><span style='color:black'>8.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>Finally,
go to the “Network Analysis” tab to perform gene-chemical interaction network
analysis based on known interactions (from experimental evidence and text
mining) included the STITCH database. Here, the target gene is EGFR. The input
chemicals are the user-added compounds with their LINCS chemical analogs based
on the MDS clustering.  </p>

<p class=MsoNormal style='margin-top:0in;margin-right:2.2pt;margin-bottom:8.0pt;
margin-left:.5in;text-indent:-.25in;line-height:107%'><span style='color:black'>9.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>In
the drop-down box, please select “Cluster Network STITCH” and click “View
Selected Cluster Network” button to see the gene-chemical interaction network
of the specific cluster number (by default, the cluster number is set at 1) of
the MDS plot from the previous tab (Figure 3). This network analysis can be
used for the interpretation and validation by mapping the added compounds or
the LINCS analogs as putative target or pathway inhibitors. For example, the
added compounds with “CHEMBL” in their names can represent direct target
inhibitors since they directly interact with the target gene EGFR, which is
indeed the case here since this demo used known EGFR active ligands from the
DUD-E database as the added compounds.  </p>

<p class=MsoNormal style='margin-bottom:7.9pt'> </p>

<p class=MsoNormal style='margin-bottom:5.8pt'> </p>

<p class=MsoNormal align=right style='margin-top:0in;margin-right:16.55pt;
margin-bottom:4.9pt;margin-left:0in;text-align:right'><img border=0 width=601
height=460 id="Picture 2008"
src="sig2Lead_version_1_1_user_manual_for_help_rev_files/image012.jpg"> </p>

<p class=MsoNormal> </p>

<p class=MsoNormal style='margin-right:2.2pt'><b>Figure 11</b>. STITCH network
analysis for EGFR, with interactions between the target gene shown as a yellow
node and candidate compounds, including the user-added compounds and their
LINCS similar analogs shown as the magenta nodes. The edges represent experimental
and text mining-based evidence of interactions retrieved from the STITCH
database. </p>

<p class=MsoNormal> </p>

<p class=MsoNormal style='margin-bottom:7.9pt'><i> </i></p>

<p class=MsoNormal> </p>

<p class=MsoNormal style='margin-bottom:7.9pt'> </p>

<p class=MsoNormal> </p>

<p class=MsoNormal style='margin-bottom:6.1pt'> </p>

<p class=MsoNormal style='margin-bottom:0in'><i><span style='font-size:9.0pt;
line-height:107%;color:#44546A'> </span></i></p>

<p class=MsoNormal>&nbsp;</p>

</div>

</body>

</html>
